scripts/run_tests.py

```diff
@@ -192,7 +192,7 @@
     "native_docker_file_image_gzip",
     "samtools_count",
     "dynamic_docker_image",
-    "ecr_docker",
+    # "ecr_docker" TODO turn back on when get credentials
```

Comment:
Would you create a ticket for this? A TODO may easily be forgotten.

Reply:
is the [IT-5064](https://jira.internal.dnanexus.com/browse/IT-5064) ticket sufficient?